### PR TITLE
Add WNP for Firefox 81 [fix #9344]

### LINKED ISSUE #9344: WNP81: Merge code by EOD 9/2
## Description

Code the WNP81 in our standard WNP template with copy from #9343 as part of epic #9342.

---

#### Success Criteria
- [ ] Coded mockups are approved by the council
- [ ] Code is merged and launched by EOD 9/2 so l10n can commence

### PR DESCRIPTION
**Pull request by craigcook**
## Description
What's new page for Firefox 81.

## Issue / Bugzilla link
#9344

## Testing
http://localhost:8000/firefox/81.0/whatsnew/all/

### CODE REVIEW

File: l10n/en/firefox/whatsnew/whatsnew-fx81.ftl [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# content blocks below hero
+whatsnew81-the-new-firefox-for-android = The NEW { -brand-name-firefox } for { -brand-name-android } is here
+
+# "Weirdo" refers to features that are unusual in a delightful way. "Indie" is short slang for independent.
+whatsnew81-our-latest-for-android-has = Our latest for { -brand-name-android } has lots of fun, smart and even weirdo features – like a moveable search bar. Because we’re an indie browser. We care about your thumbs.
+
+# "Phoning it in" is when someone doesn't fully commit to doing something and just does the minimum.
+whatsnew81-no-more-phoning-it-in = No more phoning it in with { -brand-name-safari }
+whatsnew81-attention-iphone-people = Attention { -brand-name-iphone } people: you can finally set { -brand-name-firefox } as your forever go-to, get-it-done, default browser. Make sure you’ve got the latest { -brand-name-firefox } for { -brand-name-ios }.
```
Comment (alexgibson) [inline l10n/en/firefox/whatsnew/whatsnew-fx81.ftl:25]: This feels a little misleading perhaps - from what I understand, you need the latest iOS update to set the default browser (which isn't quite out yet), and not the latest Firefox for iOS update?

File: l10n/en/firefox/whatsnew/whatsnew-fx81.ftl [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+whatsnew81-scan-this-qr-code = Scan this QR code to download
+
+# content blocks below hero
+whatsnew81-the-new-firefox-for-android = The NEW { -brand-name-firefox } for { -brand-name-android } is here
+
+# "Weirdo" refers to features that are unusual in a delightful way. "Indie" is short slang for independent.
+whatsnew81-our-latest-for-android-has = Our latest for { -brand-name-android } has lots of fun, smart and even weirdo features – like a moveable search bar. Because we’re an indie browser. We care about your thumbs.
+
+# "Phoning it in" is when someone doesn't fully commit to doing something and just does the minimum.
+whatsnew81-no-more-phoning-it-in = No more phoning it in with { -brand-name-safari }
```
Comment (alexgibson) [inline l10n/en/firefox/whatsnew/whatsnew-fx81.ftl:24]: "Phoning it in" sounds very US centric to me (it's not commonly used in the UK at least). Would this be better as an en-US specific string in its own .ftl file, and then we expose a more plain language string here in Pontoon?

File: tests/functional/firefox/whatsnew/test_whatsnew_81.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+@pytest.mark.nondestructive
+def test_send_to_device_fails_when_missing_required_fields(base_url, selenium):
+    page = FirefoxWhatsNew81Page(selenium, base_url, params='').open()
+    with pytest.raises(TimeoutException):
+        page.send_to_device.click_send()
+
+
+@pytest.mark.skip_if_not_firefox(reason='Whatsnew pages are shown to Firefox only.')
+@pytest.mark.nondestructive
+def test_get_firefox_qr_code(base_url, selenium):
```
Comment (alexgibson) [inline tests/functional/firefox/whatsnew/test_whatsnew_81.py:32]: This test is going to fail on stage / prod unless the translation for `sv-SE` is active.

**Review by alexgibson — CHANGES_REQUESTED** [review]
A couple of string questions, but otherwise this lgtm 👍

File: l10n/en/firefox/whatsnew/whatsnew-fx81.ftl (hunk 2 above)
Comment (craigcook) [reply l10n/en/firefox/whatsnew/whatsnew-fx81.ftl:24]: Agreed. I've asked for a simpler alternative.

**Review by alexgibson — APPROVED** [review]

**Comment by alexgibson** [comment]
Approved to merge once you have string updates 💯

**Pull request merged**